bin/oref0-setup.sh

```diff
@@ -989,6 +998,25 @@ if [[ $REPLY =~ ^[Yy]$ ]]; then
     echo "export API_SECRET" >> $HOME/.bash_profile
 
     echo
+    
+    #Check to see if Explorer HAT is present, and install all necessary stuff
+    if grep -a "Explorer HAT" /proc/device-tree/hat/product ; then
+        echo "Looks like you're using an Explorer HAT!"
+        echo "Making sure SPI is enabled..."
+        sed -i.bak -e "s/#dtparam=spi=on/dtparam=spi=on/" /boot/config.txt
+        echo "Enabling i2c device nodes..."
+        sed -i.bak -e "s/#dtparam=i2c_arm=on/dtparam=i2c_arm=on/" /boot/config.txt
+        egrep "^dtparam=i2c1=on" /boot/config.txt || echo "dtparam=i2c1=on,i2c1_baudrate=400000" >> /boot/config.txt
+        echo "i2c-dev" > /etc/modules-load.d/i2c.conf
+        echo "Installing socat..."
+        apt-get install socat
+        echo "Installing openaps-menu..."
+        cd $HOME/src && git clone git://github.com/openaps/openaps-menu.git || (cd openaps-menu && git checkout master && git pull)
+        cd $HOME/src/openaps-menu && sudo npm install
+        cp $HOME/src/openaps-menu/openaps-menu.service /etc/systemd/system/ && systemctl enable openaps-menu
+        cd $HOME/myopenaps && openaps alias remove battery-status && openaps alias add battery-status '! bash -c "sudo ~/src/openaps-menu/scripts/getvoltage.sh > monitor/edison-battery.json"'
```

Comment:
I think this line is causing: 
```
Created symlink /etc/systemd/system/multi-user.target.wants/openaps-menu.service → /etc/systemd/system/openaps-menu.service.
usage: openaps-alias remove [-h]
                            {autosens,bg-fresh-check,enact,format-latest-nightscout-treatments,format-ns-status,gather,get-bg,get-ns-bg,get-ns-glucose,get-settings,git,invoke,latest-ns-treatment-time,merge-pumphistory-long,mmtune,monitor-cgm,monitor-pump,monitor-xdrip,ns-loop,ns-meal-carbs,ns-temptargets,pebble,pump-loop,refresh-old-profile,refresh-old-pumphistory,refresh-old-pumphistory-24h,refresh-profile,refresh-pumphistory-24h,refresh-pumphistory-and-enact,refresh-temp-and-enact,upload,upload-ns-status,upload-recent-treatments,wait-for-long-silence,wait-for-silence}
openaps-alias remove: error: argument name: invalid choice: 'battery-status' (choose from 'autosens', 'bg-fresh-check', 'enact', 'format-latest-nightscout-treatments', 'format-ns-status', 'gather', 'get-bg', 'get-ns-bg', 'get-ns-glucose', 'get-settings', 'git', 'invoke', 'latest-ns-treatment-time', 'merge-pumphistory-long', 'mmtune', 'monitor-cgm', 'monitor-pump', 'monitor-xdrip', 'ns-loop', 'ns-meal-carbs', 'ns-temptargets', 'pebble', 'pump-loop', 'refresh-old-profile', 'refresh-old-pumphistory', 'refresh-old-pumphistory-24h', 'refresh-profile', 'refresh-pumphistory-24h', 'refresh-pumphistory-and-enact', 'refresh-temp-and-enact', 'upload', 'upload-ns-status', 'upload-recent-treatments', 'wait-for-long-silence', 'wait-for-silence')
Resetting spi_serial
```

Reply:
Not sure this line is actually needed, so that should be harmless.  We can either remove it, or change the second `&&` to a `2>/dev/null ;` to allow it to proceed to add the alias if removal fails.